CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 Changelog
 =========
 
+1.2.8
```

Comment:
Ah, I thought master is the next hotfix release... Actually @alexander-schranz has to tell how fast he needs that fix, for me the next minor release is also good 😃